extensions/trackedfile/src/main/java/io/deephaven/extensions/trackedfile/TrackedSeekableChannelsProvider.java

```diff
@@ -95,6 +92,12 @@ public Stream<URI> walk(@NotNull final URI directory) throws IOException {
         return Files.walk(Path.of(directory)).map(path -> FileUtils.convertToURI(path, false));
     }
 
+    // TODO Discuss with Ryan if I still should use this method
```

Comment:
I think it's good. Check with @abaranec if they use these APIs before changing them.